indexes/plugin.js

```diff
@@ -61,6 +61,15 @@ module.exports = function (
       }
     }
 
+    function liveStream()
+    {
```

Comment:
I'm curious why didn't the code formatter git hook apply here? It should run automatically when you commit

Reply:
Yeah that it strange, I just ran it manually and it made some minor changes. Will commit and merge.

Reply:
oh, maybe its because it is only running on staged files? I just git commit -a